.github/workflows/integration-tests.yaml

```diff
@@ -0,0 +1,46 @@
+name: Integration Tests
+on:
+  pull_request:
+    branches:
+      - master
+    types:
+      - opened
+      - reopened
+      - synchronize
+
+jobs:
+  selenium-tests:
+    name: Selenium Tests
+    runs-on: ubuntu-latest
+    env:
+      PROJECT: wordpress-plugin
+      ORGANIC_DEMO_SITE_APIKEY: ${{ secrets.ORGANIC_DEMO_SITE_API_KEY }}
+      COMPOSER_AUTH: '{"github-oauth":{"github.com":"${{ secrets.GITHUB_TOKEN }}"}}'
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Set up python
+        id: python
+        uses: actions/setup-python@v4
+        with:
+          python-version: '3.11'
+
+      - name: Install poetry
+        run: |
+          curl -sSL https://install.python-poetry.org | python3 -
+
+      - name: Configure poetry
+        run: |
+          poetry self add poetry-dotenv-plugin
+          poetry install
+
+      - name: Build the stack
+        run: |
+          mv .env.ci .env
+          composer config --global github-oauth.github.com ${{ secrets.ORGANIC_DEMO_SITE_API_KEY }}
```

Comment:
See, for instance, https://gist.github.com/nmnp/0d23e1026b54d6f5bc7caa740240a437. Without this, I was getting intermittent failures with composer installing the dependencies.

Reply:
But why site API key (secrets.ORGANIC_DEMO_SITE_API_KEY) is used here?

Reply:
🤦
Well, maybe this didn't help after all.